processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/branches.asm (1 of 19)
processing file asmFiles/branches.asm~ (2 of 19)
processing file asmFiles/count_days.asm (3 of 19)
--- memsim.hex	2015-10-21 13:27:43.269976576 -0400
+++ memcpu.hex	2015-10-21 13:27:43.255976180 -0400
@@ -50,6 +50,6 @@
 :043FFA000000001EA5
 :043FFB0000000008BA
 :043FFC00000007D0EA
-:043FFD000000016D52
-:043FFE00000007DFD9
+:043FFE000000016D51
+:043FFF00000007DFD8
 :00000001FF
processing file asmFiles/count_days.asm~ (4 of 19)
--- memsim.hex	2015-10-21 13:27:44.692016737 -0400
+++ memcpu.hex	2015-10-21 13:27:44.677016311 -0400
@@ -50,6 +50,6 @@
 :043FFA000000001EA5
 :043FFB0000000008BA
 :043FFC00000007D0EA
-:043FFD000000016D52
-:043FFE00000007DFD9
+:043FFE000000016D51
+:043FFF00000007DFD8
 :00000001FF
processing file asmFiles/fib.asm (5 of 19)
processing file asmFiles/jumpandlink.asm (6 of 19)
processing file asmFiles/jumpandlink.asm~ (7 of 19)
processing file asmFiles/mergesort.asm (8 of 19)
